src/Raven.Client/Server/Operations/ConfigureExpirationBundleOperation.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Client.Server.expiration;
+using Sparrow.Json;
+
+namespace Raven.Client.Server.Operations
+{
+    public class ConfigureExpirationBundleOperation : IServerOperation<ConfigureExpirationBundleOperationResult>
+    {
+        private ExpirationConfiguration _configuration;
+        private string _databaseName;
+
+        public ConfigureExpirationBundleOperation(ExpirationConfiguration configuration, string databaseName)
+        {
+            _configuration = configuration;
+            _databaseName = databaseName;
+        }
+        public RavenCommand<ConfigureExpirationBundleOperationResult> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+            return new ConfigureExpirationBundleCommand(_configuration, _databaseName, context);
+        }
+    }
+
+    public class ConfigureExpirationBundleCommand : RavenCommand<ConfigureExpirationBundleOperationResult>
+    {
+        private ExpirationConfiguration _configuration;
+        private readonly string _databaseName;
+        private JsonOperationContext _context;
+
+        public ConfigureExpirationBundleCommand(ExpirationConfiguration configuration, string databaseName, JsonOperationContext context)
+        {
+            _configuration = configuration;
+            _databaseName = databaseName;
+            _context = context;
+        }
+
+        public override bool IsReadRequest => false;
+
+        public override HttpRequestMessage CreateRequest(ServerNode node, out string url)
+        {
+            url = $"{node.Url}/admin/config-expiration-bundle?name={_databaseName}";
```

Comment:
No, we are defining this for a specific db, should be: `POST /databases/<db-name>/admin/expiration`